catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -82,6 +92,21 @@
 
     private static final WKTWriter WKT_WRITER = new WKTWriter();
 
+    // For queries we use repairConvexHull which my cause false positives to be returned but this
```

Comment:
Is there a place in the documentation where we can add a note or warning about this?